conftest.py

```diff
@@ -20,10 +20,16 @@
 import os
 import shutil
 import tempfile
+import warnings
 
+from keras.preprocessing.image import ImageDataGenerator
+from mxnet import gluon
 import numpy as np
 import pytest
 import requests
+import tensorflow as tf
+from torch.utils.data import DataLoader
+import torch
```

Comment:
Please don't use framework imports at the top to allow running single tests without having to install all frameworks, follow the example on `dev_1.4.0`.